x-pack/plugins/apm/public/utils/formatters/__test__/duration.test.ts

```diff
@@ -30,96 +30,6 @@ describe('duration formatters', () => {
-      expect(convertTo({ unit, microseconds: oneHourAsMicro })).toEqual({
-        unit: 'h',
-        value: '1.0',
-        formatted: '1.0 h',
-      });
-      expect(convertTo({ unit, microseconds: twoHourAsMicro })).toEqual({
-        unit: 'h',
-        value: '2.0',
-        formatted: '2.0 h',
-      });
-      expect(
-        convertTo({ unit, microseconds: null, defaultValue: '1.2' })
-      ).toEqual({ value: '1.2', formatted: '1.2' });
-    });
-
-    it('minutes', () => {
-      const unit = 'minutes';
-      const oneHourAsMicro = toMicroseconds(1, 'hours');
-      const twoHourAsMicro = toMicroseconds(2, 'hours');
-      expect(convertTo({ unit, microseconds: oneHourAsMicro })).toEqual({
-        unit: 'min',
-        value: '60.0',
-        formatted: '60.0 min',
-      });
-      expect(convertTo({ unit, microseconds: twoHourAsMicro })).toEqual({
-        unit: 'min',
-        value: '120.0',
-        formatted: '120.0 min',
-      });
-      expect(
-        convertTo({ unit, microseconds: null, defaultValue: '10' })
-      ).toEqual({ value: '10', formatted: '10' });
-    });
-
-    it('seconds', () => {
-      const unit = 'seconds';
-      const twentySecondsAsMicro = toMicroseconds(20, 'seconds');
-      const thirtyFiveSecondsAsMicro = toMicroseconds(35, 'seconds');
-      expect(convertTo({ unit, microseconds: twentySecondsAsMicro })).toEqual({
-        unit: 's',
-        value: '20.0',
-        formatted: '20.0 s',
-      });
-      expect(
-        convertTo({ unit, microseconds: thirtyFiveSecondsAsMicro })
-      ).toEqual({ unit: 's', value: '35.0', formatted: '35.0 s' });
-      expect(
-        convertTo({ unit, microseconds: null, defaultValue: '10' })
-      ).toEqual({ value: '10', formatted: '10' });
-    });
-
-    it('milliseconds', () => {
-      const unit = 'milliseconds';
-      const twentyMilliAsMicro = toMicroseconds(20, 'milliseconds');
-      const thirtyFiveMilliAsMicro = toMicroseconds(35, 'milliseconds');
-      expect(convertTo({ unit, microseconds: twentyMilliAsMicro })).toEqual({
-        unit: 'ms',
-        value: '20',
-        formatted: '20 ms',
-      });
-      expect(
-        convertTo({ unit, microseconds: thirtyFiveMilliAsMicro })
-      ).toEqual({ unit: 'ms', value: '35', formatted: '35 ms' });
-      expect(
-        convertTo({ unit, microseconds: null, defaultValue: '10' })
-      ).toEqual({ value: '10', formatted: '10' });
-    });
-
-    it('microseconds', () => {
-      const unit = 'microseconds';
-      expect(convertTo({ unit, microseconds: 20 })).toEqual({
-        unit: 'μs',
-        value: '20',
-        formatted: '20 μs',
-      });
-      expect(convertTo({ unit, microseconds: 35 })).toEqual({
-        unit: 'μs',
-        value: '35',
-        formatted: '35 μs',
-      });
-      expect(
-        convertTo({ unit, microseconds: null, defaultValue: '10' })
-      ).toEqual({ value: '10', formatted: '10' });
-    });
-  });
```

Comment:
Are these no longer needed?

Reply:
... oh, it's because `convertTo` is no longer public but only used internally?

Reply:
Yes. And we still have 100% coverage.